marshmallow/utils.py

```diff
@@ -334,16 +334,31 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
```

Comment:
I just tried my best to rebase and update the pull request. Hope it worked all out, otherwise please let me know.